src/components/Login.js

```diff
@@ -37,44 +37,54 @@ export default class Login extends Component {
   };
   render() {
     return (
-      <form
-        style={style.container}
-        onSubmit={event => this.handleSubmit(event)}
-      >
-        <h3>Login</h3>
-        <TextField
-          hintText="Enter your Email"
-          floatingLabelText="Email"
-          onChange={(event, newValue) => this.setState({ email: newValue })}
-        />
-        <br />
-        <TextField
-          type="password"
-          hintText="Enter your Password"
-          floatingLabelText="Password"
-          onChange={(event, newValue) => this.setState({ password: newValue })}
-        />
-        <br />
-        {this.state.loginMessage && (
-          <div className="alert alert-danger" role="alert">
-            <span
-              className="glyphicon glyphicon-exclamation-sign"
-              aria-hidden="true"
-            />
-            <span className="sr-only">Error:</span>
-            &nbsp;{this.state.loginMessage}{' '}
-            <a href="#" onClick={this.resetPassword} className="alert-link">
-              Forgot Password?
-            </a>
-          </div>
-        )}
-        <RaisedButton
-          label="Login"
-          primary={true}
-          style={style.raisedBtn}
-          type="submit"
-        />
-      </form>
+      <div>
+        <form
+          style={style.container}
+          onSubmit={event => this.handleSubmit(event)}
+        >
+          <h3>Login</h3>
+          <TextField
+            hintText="Enter your Email"
+            floatingLabelText="Email"
+            onChange={(event, newValue) => this.setState({ email: newValue })}
+          />
+          <br />
+          <TextField
+            type="password"
+            hintText="Enter your Password"
+            floatingLabelText="Password"
+            onChange={(event, newValue) => this.setState({ password: newValue })}
+          />
+          <br />
+          {this.state.loginMessage && (
+            <div className="alert alert-danger" role="alert">
+              <span
+                className="glyphicon glyphicon-exclamation-sign"
+                aria-hidden="true"
+              />
+              <span className="sr-only">Error:</span>
+              &nbsp;{this.state.loginMessage}{' '}
+              <a href="#" onClick={this.resetPassword} className="alert-link">
+                Forgot Password?
+              </a>
+            </div>
+          )}
+          <RaisedButton
+            label="Login"
+            primary={true}
+            style={style.raisedBtn}
+            type="submit"
+          />
+        </form>
+        <div>
+          <h5>OR</h5>
+        </div>
+        <div>
+          <p>
+            <button type="button" onClick={googleLogin}>Login with Google</button>
```

Comment:
copy the format of the RaisedButton component for better aesthetics and lose the "OR"